README.md

```diff
@@ -1,4 +1,4 @@
-# Cassandrax
+# README
```

Comment:
To make docs more comprehensible, since it already has a `Cassandrax` page as the `index` page. If we keep `Cassandrax` as the title of the README, it will have 2 `Cassandrax` pages.

Reply:
This is also the readme of the repository, I'd like to keep the name of the library as its header.